### PR TITLE
add Opensearch config parameters override samples

### PR DESCRIPTION
**Pull request by AnilMaktala**
#### Description of changes:
Added Opensearch Override samples
#### Related GitHub issue #, if available:
#3963 
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

This PR affects which product(s) (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by renebrandel — APPROVED** [review]

File: src/fragments/start/getting-started/android/integrate.mdx [hunk 1]
```diff
@@ -476,36 +476,36 @@ You may want to change the contents of a record. Below, we'll query for a record
...
-                  .build();
-              Amplify.DataStore.save(updatedTodo,
-                  updated -> Log.i("Tutorial", "Updated item: " + updatedTodo.getName(),
-                  failure -> Log.e("Tutorial", "Update failed.", failure)
-              );
-          }
-      },
-      failure -> Log.e("Tutorial", "Query failed.", failure)
-  );
+    Amplify.DataStore.query(Todo.class, Where.matches(Todo.NAME.eq("Finish quarterly taxes")),
```
Comment (josefaidt) [inline src/fragments/start/getting-started/android/integrate.mdx:479]: were these changes intentionally captured in this PR?

File: src/pages/cli/graphql/override.mdx [hunk 2]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.opensearch.OpenSearchStreamingLambdaFunction.FunctionName = 'CustomFunctionName';
+}
+
+```
+
+### Example - Configure OpenSearch instance version
+
+Override the elasticsearchVersion in the OpenSearch domain created by @searchable
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:135]:
```suggestion
Override the `elasticsearchVersion` in the OpenSearch domain created by `@searchable`
```

File: src/pages/cli/graphql/override.mdx [hunk 3]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.opensearch.OpenSearchDomain.elasticsearchVersion = 'OpenSearch_1.3';
+}
+```
+### Example - Configure OpenSearch instance type
+
+Override the type of instance launched into the OpenSearch domain created by @searchable
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:146]:
```suggestion
Override the type of instance launched into the OpenSearch domain created by `@searchable`
```

File: src/pages/cli/graphql/override.mdx [hunk 4]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+    resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
+        ...resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig,
+        instanceType: "m5.large.search"
+    }
+   
+}
+```
+### Example - Configure OpenSearch instance count
+Override the number of instances launched into the OpenSearch domain created by @searchable
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:160]:
```suggestion
Override the number of instances launched into the OpenSearch domain created by `@searchable`
```

File: src/pages/cli/graphql/override.mdx [hunk 5]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+    resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
+        ...resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig,
+        instanceCount: 2
+    }
+   
+}
+```
+
+### Example - Configure OpenSearch EBS volume size
+Override the amount of disk space allocated to each instance in the OpenSearch domain created by @searchable
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:175]:
```suggestion
Override the amount of disk space allocated to each instance in the OpenSearch domain created by `@searchable`
```

File: src/pages/cli/graphql/override.mdx [hunk 6]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+    resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
+        ...resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig,
+        instanceType: "m5.large.search"
+    }
+
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:156]:
```suggestion
```

File: src/pages/cli/graphql/override.mdx [hunk 7]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+      resources.opensearch.OpenSearchDomain.ebsOptions={
+        ...resources.opensearch.OpenSearchDomain.ebsOptions,
+        volumeSize:10
+      }
+
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:185]:
```suggestion
```

File: src/pages/cli/graphql/override.mdx [hunk 8]
```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
...
+}
+```
+### Example - Configure OpenSearch instance count
+Override the number of instances launched into the OpenSearch domain created by @searchable
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+    resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
```
Comment (josefaidt) [inline src/pages/cli/graphql/override.mdx:166]: the tab spacing is different here than in the previous examples where it appears to be using 2 spaces https://github.com/aws-amplify/docs/pull/5291/files#diff-edcb511e6975a1b33dbcf4d4d0f70e70f3465fb841c01016f01988df6d151fb3R141

**Review by josefaidt — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Review by sundersc — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**